ui/resolution_selection.js

```diff
@@ -35,6 +35,7 @@ shaka.ui.ResolutionSelection = class extends shaka.ui.SettingsMenu {
 
     this.button.classList.add('shaka-resolution-button');
     this.menu.classList.add('shaka-resolutions');
+    this.flag = 0;
```

Comment:
Use an argument for this, this is exactly what arguments are made for.  You can use a default argument to avoid updating a lot of callsites, but it is better to just update all of them instead of having a poorly named field to hold arguments.